server-1937-review-support-bundle | added support bundle steps (#7045)
circleci/circleci-docs: jekyll/_cci2/server/operator/troubleshooting-and-support.adoc

toc::[]

[#generate-support-bundle]
== Generate Support Bundle
A support bundle is used by CircleCI engineers to diagnose and fix any issues you are experiencing. They are typically requested when you open a ticket.

> To generate a support bundle, You need to follow below steps - 

Reviewer: ```suggestion
To generate a support bundle, follow the steps below.
```

=== Prerequisites

> . First, make sure circleci-server is deployed and you have access to the cluster/namespace through kubectl.

Reviewer: @atulsingh0 can we add some commands here that people can copy to make this really straightforward for people?

Author: sure

. Next, install link:https://krew.sigs.k8s.io/docs/user-guide/setup/install/[krew].

> . Install preflight and support bundle to your local development machine.

Reviewer: I'm not clear on this step - what's 'preflight and support bundle' in this context?
